README.md

```diff
@@ -49,90 +37,120 @@ app.use(errorHandler({
 app.listen(3000);
 ```
 
-In LoopBack applications, add the following entry to your
-`server/middleware.json` file.
+In LoopBack applications, add the following entry to `server/middleware.json`:
 
 ```json
 {
   "final:after": {
     "strong-error-handler": {
       "params": {
          "debug": false,
-         "log": true,
+         "log": true
        }
     }
   }
 }
 ```
 
-## Content Type
-
-Depending on the request header's `Accepts`, response will be returned in
- the corresponding content-type, current supported types include:
-- JSON (`json`/`application/json`)
-- HTML (`html`/`text/html`)
+The above configuration will log errors to the server console, but not return stack traces in HTTP responses.
+For details on configuration options, see below.
 
-*There are plans to support other formats such as Text and XML.*
+### Response format and content type
 
-## Options
+The `strong-error-handler` package supports HTML and JSON responses:
 
-#### debug
+- When the object is a standard Error object, it returns the string provided by the stack property in HTML/text
+  responses.
+- When the object is a non-Error object, it returns the result of `util.inspect` in HTML/text responses.
+- For JSON responses, the result is an object with all enumerable properties from the object in the response.
 
-`boolean`, defaults to `false`.
+The content type of the response depends on the request's `Accepts` header.
 
-If you need to set the environment to development mode, you will need to change the value to `true`.
+-  For Accepts header `json` or `application/json`, the response content type is JSON.
+-  For Accepts header `html` or `text/html`, the response content type is HTML.
 
-When enabled, HTTP responses include all error properties, including
-sensitive data such as file paths, URLs and stack traces. *See Examples below*
+*There are plans to support other formats such as Text and XML.*
 
-#### log
+## Options
 
-`boolean`, defaults to `true`.
+| Option | Type | Default | Description |
+| ---- | ---- | ---- | ---- |
+| debug | Boolean&nbsp;&nbsp;&nbsp; | `false` | If `true`, HTTP responses include all error properties, including sensitive data such as file paths, URLs and stack traces. *See Examples below* |
```

Comment:
`*See Examples below*` - is it intentional that the whole sentence is in italics, and there is no dot at the end? Is it worth changing "Examples" into a link pointing to the right section?